packages/waku/src/router/client.ts

```diff
@@ -144,6 +144,7 @@ export type LinkProps = {
   notPending?: ReactNode;
   children: ReactNode;
   unstable_prefetchOnEnter?: boolean;
+  unstable_prefetchOnView?: boolean;
```

Comment:
As long as it's marked `unstable`, it feels safe to me. We may revisit "prefetching" thing later.